specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -3780,15 +3780,15 @@
       },
       "put": {
         "x-ms-examples": {
-          "Upsert the Threat Intelligence": {
+          "Update a threat Intelligence indicator": {
             "$ref": "./examples/threatintelligence/UpsertThreatIntelligence.json"
           }
         },
         "tags": [
           "ThreatIntelligence"
         ],
-        "description": "Upsert a threat intelligence.",
-        "operationId": "ThreatIntelligenceIndicatorUpsert_Create",
+        "description": "Update a threat Intelligence indicator.",
+        "operationId": "ThreatIntelligenceIndicator_Create",
```

Comment:
ThreatIntelligenceIndicator_Create? Not Update, or UpdateIndicator?

Reply:
This is strange. We have an operation called ThreatIntelligenceIndicator_CreateIndicator and another one ThreatIntelligenceIndicator_Create. I believe this should be resolved somehow... is it not?

Reply:
"Put" requires _Create